gettsim/data/soz_vers_beitr.yaml

```diff
@@ -1,1192 +1,749 @@
-      note: https://www.bundesregierung.de/Content/DE/Artikel/2016/10/2016-10-12-bemessunggrenzen-sozialversicherung.html
-    2018-01-01:
-      value: 4425
-      note:
-    2019-01-01:
-      value: 4537.5
-      note: https://www.bundesregierung.de/breg-de/suche eue-beitragsbemessungsgrenzen-1535600
-    2020-01-01:
-      value: 4687.5
-      note: §4 V. v. 17.12.2019 BGBl. I S. 2848
-kvmaxekw:
-  name:
-    de: Beitragsbemessungsgrenze in der GKV und PV (West)
-    en: Income Threshold for Health and Care Insurance. West Germany
+    de: Beitragssätze zu den gesetzlichen Sozialversicherungen
+    en: Contribution rate to statutory social insurances
   description:
     de:
     en:
-  values:
-    1984-01-01:
-      value: 1994
-      note:
-    1985-01-01:
-      value: 2071
-      note:
-    1986-01-01:
-      value: 2147
-      note:
-    1987-01-01:
-      value: 2186
-      note: .
-    1988-01-01:
-      value: 2301
-      note: .
-    1989-01-01:
-      value: 2339
-      note:
-    1990-01-01:
-      value: 2416
-      note: .
-    1991-01-01:
-      value: 2493
-      note:
-    1992-01-01:
-      value: 2608
-      note: .
-    1993-01-01:
-      value: 2761
-      note: .
-    1994-01-01:
-      value: 2914
-      note:
-    1995-01-01:
-      value: 2991
-      note: .
-    1996-01-01:
-      value: 3068
-      note: .
-    1997-01-01:
-      value: 3144
-      note: .
-    1998-01-01:
-      value: 3221
-      note:
-    1999-01-01:
-      value: 3259
-      note:
-    2000-01-01:
-      value: 3298
-      note: .
-    2001-01-01:
-      value: 3336
-      note: .
-    2002-01-01:
-      value: 3375
-      note:
-    2003-01-01:
-      value: 3450
-      note:
-    2004-01-01:
-      value: 3488
-      note:
-    2005-01-01:
-      value: 3525
-      note:
-    2006-01-01:
-      value: 3562.5
-      note:
-    2008-01-01:
-      value: 3600
-      note:
-    2009-01-01:
-      value: 3675
-      note:
-    2010-01-01:
-      value: 3750
-      note:
-    2011-01-01:
-      value: 3712.5
-      note:
-    2012-01-01:
-      value: 3825
-      note:
-    2013-01-01:
-      value: 3937.5
-      note:
-    2014-01-01:
-      value: 4050
-      note: http://www.bundesregierung.de/ContentArchiv/DE/Archiv17/Artikel/2013/10/2013-10-16-rechengroessen-sozialversicherung.html
-    2015-01-01:
-      value: 4125
-      note:
-    2016-01-01:
-      value: 4237.5
-      note: https://www.bundesregierung.de/Content/DE/Artikel/2015/10/2015-10-14-sozialversicherung.html
-    2017-01-01:
-      value: 4350
-      note: https://www.bundesregierung.de/Content/DE/Artikel/2016/10/2016-10-12-bemessunggrenzen-sozialversicherung.html
-    2018-01-01:
-      value: 4425
-      note:
-    2019-01-01:
-      value: 4537.5
-      note: https://www.bundesregierung.de/breg-de/suche eue-beitragsbemessungsgrenzen-1535600
-    2020-01-01:
-      value: 4687.5
-      note: §4 V. v. 17.12.2019 BGBl. I S. 2848
-alvbs:
+  1984-01-01:
+    ges_krankv:
```

Comment:
Ordering them the other way round (first the branch (health, unemployment, ...), then the date) occurs to me more natural

Reply:
Then they would all be single variables. I think that's a first way to regroup them. If you think of a general better structure, please suggest and we discuss in less busy time! I think this way it is more comprehensive than single variables.